packages/terra-alert/docs/MoreInformation.md

```diff
@@ -0,0 +1,150 @@
+|type  |string  |optional  |'alert' |The type of alert to be rendered. One of Alert.Opts.Types.ALERT, Alert.Opts.Types.ERROR, Alert.Opts.Types.WARNING, Alert.Opts.Types.ADVISORY, Alert.Opts.Types.INFO, Alert.Opts.Types.SUCCESS, Alert.Opts.Types.GAP_CHECKING, Alert.Opts.Types.OUTSIDE_RECORDS, Alert.Opts.Types.CUSTOM. Use the Alert.Opts.Types attribute of the Alert component for access to these type strings.|
+
+## Examples
+
+### Imports
+When importing the Alert component, it includes the Types object which contains a series of string constants that can be used for setting the 'type' prop for the component as shown in the examples below.
+```jsx
+import React from 'react';
+import Alert from 'terra-alert';
+```
+
+### Examples
+__________
+Basic Alert component of type 'alert' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.ALERT} >This is an alert</Alert>
+```
+![Image](images/basicAlert.png?raw=true)
+__________
+Basic Alert component of type 'error' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.ERROR} >This is an error</Alert>
+```
+![Image](images/basicError.png?raw=true)
+__________
+Basic Alert component of type 'warning' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.WARNING} >This is a warning</Alert>
+```
+![Image](images/basicWarning.png?raw=true)
+__________
+Basic Alert component of type 'advisory' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.ADVISORY} >This is an advisory alert</Alert>
+```
+![Image](images/basicAdvisory.png?raw=true)
+__________
+Basic Alert component of type 'info' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.INFO} >This is an information alert</Alert>
+```
+![Image](images/basicInformation.png?raw=true)
+__________
+Basic Alert component of type 'success' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.SUCCESS} >This is a success alert</Alert>
+```
+![Image](images/basicSuccess.png?raw=true)
+__________
+Basic Alert component of type 'gap-checking' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.GAP_CHECKING} >This is a gap-checking alert</Alert>
+```
+![Image](images/basicGapChecking.png?raw=true)
+__________
+Basic Alert component of type 'outside records' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.OUTSIDE_RECORDS} >This is an outside records alert</Alert>
+```
+![Image](images/basicOutsideRecords.png?raw=true)
+__________
+Alert of type 'custom' with custom title, status color, and icon, and includes HTML in message content.
+```jsx
+<Alert type={Alert.Opts.Types.CUSTOM} 
+       title="Help!" 
+       customStatusColor="orange" 
```

Comment:
I don't think that separating CustomAlert into it's own component would rectify this. I'm not sure it's worth the work to do it.